net/FlatBuffers/Table.cs

```diff
@@ -83,15 +84,15 @@ protected int __vector(int offset)
         }
 
         // Initialize any Table-derived type to point to the union at the given offset.
-        protected TTable __union<TTable>(TTable t, int offset) where TTable : Table
+        public T __union<T>(int offset) where T : struct, Accessor
```

Comment:
Thanks for testing. Decode has the biggest speedup, but isn't that essentially a no-op? Isn't all the object access in Traverse? If so, the speedup in Traverse is not particularly impressive, though it may also mean it wasn't really bottlenecked by accessor creation to start with (was your old code re-using accessor objects?)

Yes, it breaks the API somewhat, which is unfortunate, but I need to represent the optional nature of things somehow, and felt that doing so with Nullable was the most idiomatic. I am not sure why they chose to require the .Value accessor, but so be it.

Yes, AOT will likely be awesome. Never been a fan of JITs and VMs :)